Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -290,17 +292,15 @@ private void ExecuteSourceOperationMoveOrRename()
                 _result.ErrorMessages.Add($"Failure in source operation: {msg}");
             }
 
-            var destFileName = moveToPath.Contains('/')
-                ? moveToPath + "/" + SourceFile.Name
-                : Path.Combine(moveToPath, SourceFile.Name);
+            var destFileName = Path.Combine(moveToPath, SourceFile.Name);
 
             try { File.Move(filePath, destFileName); }
-            catch (Exception ex) { throw new Exception($"Failure in source operation: {ex.Message}"); }
+            catch { throw new Exception($"Failure in source operation: File {Path.GetFileName(destFileName)} exists in move to directory."); }
```

Comment:
I think we should include the actual exception that happened as an inner exception here. Otherwise it might be hard to understand the problem sometimes.

Reply:
Fixed by returning the old implementation.